### DISCUSSION TITLE
Consulting

### DISCUSSION THREAD

**Discussion by mickdelaney** [elsa-workflows/elsa-core#804] [General]
Hi,

Are any of the core committers available to do a bit of consulting ? 
Essentially some project planning for a Elsa 2x project to be kicked off in April.

I'm based in London, UK. 

Regards

**Comment by craigfowler** [comment]
@mickdelaney thanks for asking.  I'm personally a bit too heavily-booked up to take on any extra work.  @sfmskywalker might be interested?

Off-topic, I just looked up Elevate Direct and noticed you're in Hatton Garden.  I'm usually based only a few mins walk away, on St John's Street.  Although, I haven't actually been to that office in over a year now because of the pandemic.

**Reply by mickdelaney** [reply]
yeah, we must meet up for a beer when things start getting back to normal ;-)

**Reply by mickdelaney** [reply]
hey @craigfowler are you still based St John's Street ? 
I'm getting a new office by Old Street roundabout in the next week or so.

**Reply by craigfowler** [reply]
@mickdelaney I am for the time being, but in approx a month we move as well; only a few hundred metres down the road.  We'll be on Great Sutton Street, pretty much opposite The Slaughtered Lamb pub.  We did switch to hybrid working post-pandemic though.  I'm usually only in the office on Mondays and Wednesdays now, unless there's exceptional circumstances (EG: rail strikes will cause me to switch days in order to dodge them).

**Reply by mickdelaney** [reply]
ah great pub !!
many a drunken developer debate has been had there ;-)
yeah, rail strikes dont have the same logic behind them now everyone can work from home LOL

**Comment by mickdelaney** [comment]
I'm very flexible on time & hours etc. At this stage I just want to get off in the right direction. 
I dont imagine it will require a huge amount of time.

**Reply by sfmskywalker** [reply]
I’m always happy to get on a call and talk about Elsa. You can reach me on Skype: sfmskywalker.
